### ISSUE TITLE
key error printing a Tag object

### ISSUE THREAD

**Issue by ghost** [shaldengeki/albatross#5]
https://github.com/shaldengeki/albatross/blob/master/albatross/tag.py#L62

should be

```
  "\n".join([staff['user'].name + ": " + staff['role'] for staff in self.staff]),
```

**Comment by shaldengeki** [comment]
Good catch! Sorry I haven't responded till now, I swear life has been crazy busy the past few days. Will have a fix committed for this tomorrow night, if you can wait that long.
